src/compression.jl

```diff
@@ -50,6 +83,9 @@ function TranscodingStreams.initialize(codec::ZstdCompressor)
     if iserror(code)
         zstderror(codec.cstream, code)
     end
+    codec.cstream.ibuffer.src = C_NULL
+    codec.cstream.ibuffer.size = 0
+    codec.cstream.ibuffer.pos = 0
```

Comment:
Sure. What public API uses startproc?

Reply:
If a previous call to `transcode` is interrupted, `startproc` should reset the state of the codec so it can start a new frame.
https://juliaio.github.io/TranscodingStreams.jl/stable/reference/#TranscodingStreams.startproc

Reply:
It gets used in https://github.com/JuliaIO/TranscodingStreams.jl/blob/7893d7463bf50577d3b2c3705851b4322ad21ccd/src/stream.jl#L784 and https://github.com/JuliaIO/TranscodingStreams.jl/blob/7893d7463bf50577d3b2c3705851b4322ad21ccd/src/transcode.jl#L147

Reply:
`startproc` calls `reset!(::Cstream, ...)`.

With https://github.com/JuliaIO/CodecZstd.jl/pull/52/commits/800c03c4f5051d5b84590e0d8fd47a16afa5c135, `reset!(::Cstream, ...)` now calls `reset!` on the input and output buffers.